executor.py

```diff
@@ -76,8 +76,10 @@ def search(
             else self._match_args
         )
 
+        traversal_right = parameters.get('traversal_right', '@r')
+        traversal_left = parameters.get('traversal_left', '@r')
         match_args = SimpleIndexer._filter_match_params(docs, match_args)
-        docs.match(self._index, **match_args)
+        docs[traversal_left].match(self._index[traversal_right], **match_args)
```

Comment:
The old way of doing it it to pass by match_args : https://github.com/jina-ai/executor-simpleindexer/tree/v0.11 It is common to a lot of our executor.

The way is the code now is that is merge match_args with the parameters so it should be 
```python
traversal_right = match_args.get('traversal_right', '@r')
traversal_left = match_args.get('traversal_left', '@r')
```

Reply:
But it's not a thing anymore in `match()`. See [here](https://docarray.jina.ai/api/docarray.array.mixins.match/#docarray.array.mixins.match.MatchMixin.match) So it shouln't be part of match_args

Reply:
It was more to keep old code working but you are right then